packages/synapse-react-client/src/lib/utils/theme/useTheme.tsx

```diff
@@ -3,8 +3,11 @@ import { ThemeProvider as MuiThemeProvider } from '@mui/material/styles'
 import { merge } from 'lodash-es'
 import React, { useMemo } from 'react'
 import defaultMuiTheme from './DefaultTheme'
+import type { PartialDeep } from 'type-fest'
 
-export function mergeTheme(themeOverrides: ThemeOptions): ThemeOptions {
+export function mergeTheme(
+  themeOverrides: PartialDeep<ThemeOptions>,
+): ThemeOptions {
```

Comment:
No need to supply an entire theme. Use this `PartialDeep` utility to mark all fields and those fields' subfields as optional.

We have a `DeepPartial` utility, but I ended up running into recursive type limit issues using that, but not with this utility type.